core/src/main/java/com/chopshop166/chopshoplib/motors/CSSparkFlex.java

```diff
@@ -20,7 +20,20 @@ public class CSSparkFlex extends CSSpark {
      *        to the Red and Black terminals only.
      */
     public CSSparkFlex(final int deviceID, final MotorType type) {
```

Comment:
We should add another constructor that defaults the type to `MotorType.kBrushless`.